src/components/chatbot.tsx

```diff
@@ -114,7 +102,7 @@ export const Chatbot: React.FC = props => {
       <StyledChatbot
         {...props}
         steps={transformedSteps}
-        handleEnd={handleEnd}
+        handleEnd={scrollToLastBotMessage}
```

Comment:
could we use a ref instead of doing document.querySelectorAll ?

Reply:
Can we get a hook to the content view? I didn't see a way